README.md

```diff
@@ -126,6 +126,32 @@ createP.then(function(response) {
 `response` is a JSON object containing the FaunaDB response. See the JSDocs for
 `faunadb.Client`.
 
+#### ES6 modules
+
+This is recommended way to inject faunadb-js. Many build tools supports three shaking which relies on the static structure of ES6 module syntax, i.e. `import` and `export`.
+
+```
+import { Client, query } from 'faunadb'
+
+const client =new Client({secret: 'your_key_here'})
+client.query(query.Now()).then(console.info)
```

Comment:
```suggestion
import { Client, q } from 'faunadb'
const client = new Client({secret: 'your_secret_here'})
client.query(q.Now()).then(console.info)
```

All of our documented examples use `q` as the namespace for FQL functions. If we don't make that change, the `client.query(query.Now())` can confuse newbies.

Reply:
we can update it it
```
import {query as q} from 'faunadb'
```
it's the equivalent to what we currently have
```
const faunadb = require('faunadb')
const q = faunadb.query;
```
but everyone familiar with both ecma script modules and commonjs modules